{{ cookiecutter.__package_name_kebab_case }}/README.md

```diff
@@ -30,6 +31,7 @@ To serve this app, run `docker compose up app` and open [localhost:8000](http://
       UseKeychain yes
     EOF
     ```
+    
 {%- if cookiecutter.private_package_repository_name %}
```

Comment:
This newline will be removed by the `{%- if cookiecutter.private_package_repository_name %}` line, so we instead need to insert it below the if:
```suggestion
{%- if cookiecutter.private_package_repository_name %}

```

We also need to a newline on line 46 in case the if statement is not true.